src/components/Personalization/constants/eventType.js

```diff
@@ -12,3 +12,7 @@ governing permissions and limitations under the License.
 
 export const DISPLAY = "decisioning.propositionDisplay";
 export const INTERACT = "decisioning.propositionInteract";
+export const EventType = {
+  DISPLAY: "display",
```

Comment:
Why not use the full event types `decisioning.proposition...`?

Reply:
@jfkhoury, using the existing eventType field with `decisioning.proposition...` isn't flexible enough for all customer use cases, since the existing `eventType` lives on the event and not the proposition.  [This wiki ](https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=CJM&title=Proposition+Event+Types#PropositionEventTypes-Problem) describes the problem and why both Target and AJO need something like propositionEventType.